src/applications/edu-benefits/tests/feedback-tool/01.feedback-tool-keyboard.e2e.spec.js

```diff
@@ -0,0 +1,224 @@
+const E2eHelpers = require('../../../../platform/testing/e2e/helpers');
+const Timeouts = require('../../../../platform/testing/e2e/timeouts');
+const PageHelpers = require('./feedback-tool-helpers');
+const FormsTestHelpers = require('../../../../platform/testing/e2e/form-helpers');
+const manifest = require('../../feedback-tool/manifest.json');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  const { ARROW_DOWN, ARROW_UP, ENTER, TAB } = client.Keys;
+
+  PageHelpers.initApplicationSubmitMock();
+
+  client
+    .openUrl(`${E2eHelpers.baseUrl}${manifest.rootUrl}`)
+    .waitForElementVisible('body', Timeouts.slow)
+    .waitForElementVisible('.schemaform-title', Timeouts.slow)
+    .click('.schemaform-start-button');
+
+  E2eHelpers.overrideVetsGovApi(client);
+  FormsTestHelpers.overrideFormsScrolling(client);
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+  // Evaluate the radio buttons for whom to submit this feedback
+  client
+    .keys(TAB)
+    .allyEvaluateRadioButtons(
+      [
+        'input#root_onBehalfOf_0',
+        'input#root_onBehalfOf_1',
+        'input#root_onBehalfOf_2',
+      ],
```

Comment:
I'm a bit torn.  On the one hand, I see the value in being specific with each selector so it is clear exactly what the helper will look at.  However, I'm also wondering if it would be a better experience to just specify the first radio button in a group (or specify the container) and have the helper take care of finding the sibling inputs, like this: 

```js
    .allyEvaluateRadioButtons(
      'div#button-group',
      ARROW_UP,
      true,
    );
```

 I say this with no idea of what the DOM looks like though, so it may not be a good idea.

Reply:
@bkjohnson This gives me an idea: https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByTagName :)